design-library/src/components/BccHeader/BccHeader.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+type Props = {
+  overline?: string;
+  title?: string;
+  underline?: string;
+};
+
+withDefaults(defineProps<Props>(), {});
+</script>
+
+<template>
+  <div>
+    <p class="text-caption text-secondary">{{ overline }}</p>
```

Comment:
Perhaps add `v-if`s to the wrapping elements for optional properties, so the markup is not rendered (unless that is what you want).